src/test/java/pages/WikipediaPage.java

```diff
@@ -0,0 +1,44 @@
+package pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import ru.yandex.qatools.ashot.AShot;
+import ru.yandex.qatools.ashot.Screenshot;
+import ru.yandex.qatools.ashot.shooting.ShootingStrategies;
+
+import javax.imageio.ImageIO;
+import java.io.File;
+import java.io.IOException;
+
+public class WikipediaPage extends AbstractPage {
+
+    private static final String WIKIPEDIA_URL = "https://en.wikipedia.org";
+
+    public WikipediaPage(WebDriver driver) {
```

Comment:
Please move constructor after all variables declaration

Reply:
Thank you for pointing, please see in the following commit.